cvcl/views/AddUser.py

```diff
@@ -10,11 +10,14 @@
 from ..models import User
 
 
-class DataView(FormView):
+class AddStudent(FormView):
     template_name = 'fileUpload.html'
     form_class = csvForm
-    success_url = '/add_students'
 
     def form_valid(self, form):
-        form.process_data()
+        course_id = self.kwargs['pk']
+        form.process_data(course_id)
         return super().form_valid(form)
+
+    def get_success_url(self):
+        return '/courses/{0}'.format(self.kwargs['pk'])
```

Comment:
This is horrible practice.
Read documentation.
https://github.com/DSpeichert/cloudvcl/blob/master/cvcl/views/environment.py#L57-L58